### PR TITLE
Stablize FPS

### PR DESCRIPTION
**Pull request by JiangYizhou**
Because type of time was float which is not precise enough,
 elapse was 0 between 2 frames when rendering small amount of
 fishes, thus, the value was wrong. So in this patch, float is
 replaced by double to calculate fps. In addition, fps is
 calculated by average of every 50ms to eliminate divisor is
 almost 0.

### CODE REVIEW

File: src/common/FPSTimer.cpp [hunk 1]
```diff
@@ -33,8 +33,8 @@ void FPSTimer::update(float elapsedTime, float renderingTime, int logCount)
         mTimeTableCursor = 0;
     }
 
-    mInstantaneousFPS = floor(1.0f / elapsedTime + 0.5f);
-    mAverageFPS = floor((1.0f / (mTotalTime / static_cast<float>(NUM_FRAMES_TO_AVERAGE))) + 0.5f);
+    mInstantaneousFPS = floor(1.0 / elapsedTime + 0.5);
+    mAverageFPS = floor((1.0 / (mTotalTime / static_cast<float>(NUM_FRAMES_TO_AVERAGE))) + 0.5);
```
Comment (gyagp) [inline src/common/FPSTimer.cpp]: Do we need the static_cast<float> here?

File: src/aquarium-optimized/Aquarium.cpp [hunk 2]
```diff
@@ -351,9 +351,9 @@ bool Aquarium::init(int argc, char **argv)
     mContext->showWindow();
 
 #ifdef _WIN32
-    g.start = GetTickCount64() / 1000.0f;
+    g.start = GetTickCount64() / 1000.0;
 #else
-    g.start   = clock() / 1000000.0f;
+    g.start    = clock() / 1000000.0;
```
Comment (gyagp) [inline src/aquarium-optimized/Aquarium.cpp]: unnecessary spaces before '='.

File: src/common/FPSTimer.cpp [hunk 3]
```diff
@@ -13,16 +13,16 @@
...
       mTimeTableCursor(0),
       mRecordFpsFrequencyCursor(0),
-      mInstantaneousFPS(0.0f),
-      mAverageFPS(0.0f)
+      mInstantaneousFPS(0.0),
+      mAverageFPS(0.0)
 {}
 
-void FPSTimer::update(float elapsedTime, float renderingTime, int logCount)
+void FPSTimer::update(double elapsedTime, double renderingTime, int logCount)
```
Comment (gyagp) [inline src/common/FPSTimer.cpp]: If we can trigger the update every several seconds, that would be more reasonable.

File: src/common/FPSTimer.cpp (hunk 1 above)
Comment (JiangYizhou) [reply src/common/FPSTimer.cpp]: The algorithm is revised to avg of every 50ms, PTAL. Thanks.

**Pull request merged**